src/jpcsp/MainGUI.java

```diff
@@ -136,8 +136,28 @@
     private List<RecentElement> recentUMD = new LinkedList<RecentElement>();
     private List<RecentElement> recentFile = new LinkedList<RecentElement>();
     private final static String[] userDir = {
+        "MP_ROOT/100ANV01",
+        "MP_ROOT/100MNV01",

```

Comment:
MP_ROOT should be under ms0, right?

Reply:
@gid15 You are right, my brain was confused.